packages/utils/src/lib/throttle.ts

```diff
@@ -0,0 +1,96 @@
+const isTest = () =>
+	typeof process !== 'undefined' &&
+	process.env.NODE_ENV === 'test' &&
+	// @ts-expect-error
+	!globalThis.__FORCE_RAF_IN_TESTS__
+
+const fpsQueue: Array<() => void> = []
+const targetFps = 60
+const targetTimePerFrame = 1000 / targetFps
+let frame: number | undefined
+let time = 0
+let last = 0
+
+const tick = () => {
+	const queue = fpsQueue.splice(0, fpsQueue.length)
+	for (const fn of queue) {
+		fn()
+	}
+}
+
+function fps() {
```

Comment:
extremely nonblocking nitpick:

I found the naming here a little odd. I'd personally change `tick` to `flush` and change `fps` to `tick`. For me a `tick` is some kind of regularly scheduled event for managing asynchronous execution of other things, which is kinda what your `fps` function is doing. 

I dunno, could just be me though 🤷🏼

Reply:
Proposed names definitely make more sense to me, thanks for the suggestion.